### PR TITLE
Update tracing_subsciber to 0.3 which uses 'time' instead of 'chrono'

### PR DESCRIPTION
**Pull request by makr11st**
Signed-off-by: Lukasz Woznicki <lukasz.woznicki@softwareag.com>

## Proposed changes

Use latest `tracing_subscriber` which has moved to crate `time`.

## Types of changes

What types of changes does your code introduce to `thin-edge.io`?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (general improvements like code refactoring that doesn't explicitly fix a bug or add any new functionality)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://github.com/thin-edge/thin-edge.io/blob/main/CONTRIBUTOR-LICENSE-AGREEMENT.md) (in all commits with git commit -s)
- [x] I ran `cargo fmt` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [x] I used `cargo clippy` as mentioned in [CODING_GUIDELINES](https://github.com/thin-edge/thin-edge.io/blob/main/CODING_GUIDELINES.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by PradeepKiruvale — APPROVED** [review]
LGTM, however, there are Clippy warnings.

File: plugins/log_request_plugin/src/main.rs [hunk 1]
```diff
@@ -71,13 +71,3 @@ async fn main() -> Result<(), anyhow::Error> {
...
     Ok(())
 }
-
-/// does a one second pause.
-///
-/// NOTE: this is a quick-fix to enable step 4. in main to be executed.
-/// if `do_one_second_pause()` is not called, step 4. in main
-/// does not get triggered.
-// see #846
-async fn do_one_second_pause() {
```
Comment (TheNeikos) [inline plugins/log_request_plugin/src/main.rs:81]: I am a bit confused with the name of the PR and the actual changes here?

Comment (makr11st) [reply plugins/log_request_plugin/src/main.rs:81]: Just a mixed up commits

**Review by matthiasbeyer — APPROVED** [review]
LGTM

**Comment by matthiasbeyer** [comment]
@makr11st there are merge conflicts, can you rebase this on latest `main`, please? :smiley:

**Pull request merged**